src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -29,6 +29,20 @@ public override Task<JObject> GetAdminSitesBySiteNameAsync(string stampName, str
             throw new NotImplementedException();
         }
 
+        public override async Task<JArray> GetAdminSitesAsync(string siteName)
+        {
+            if (string.IsNullOrWhiteSpace(siteName))
+            {
+                throw new ArgumentNullException(nameof(siteName));    
+            }
+
+            var path = $"sites/{siteName}/adminsites";
+            
+            var response = await GetObserverResource(path);
```

Comment:
Don't we need to check the response here?

Reply:
Yes, good catch